interface/JECKLUBinterface.h

```diff
@@ -41,7 +41,7 @@ typedef std::map<std::string, std::unique_ptr<JetCorrectionUncertainty>> myJECMa
 class JECKLUBinterface {
 
 public:
-  JECKLUBinterface (std::string year, std::string version);
+  JECKLUBinterface (std::string year, bool isPostVFP = false);
```

Comment:
Why removing the version? I understand that **for the moment** there is only one version of the JECs, but maybe in the future there will be more.

Reply:
I guess that if a new versions comes out we can just replace the name of the txt files.